internal/hcsoci/create.go

```diff
@@ -26,7 +27,7 @@ import (
 )
 
 var (
-	lcowRootInUVM = "/run/gcs/c/%s"
+	lcowRootInUVM = guestpath.LCOWRootPrefixInUVM + "/%s"
 	wcowRootInUVM = `C:\c\%s`
```

Comment:
This should reference the value in `guestpath` too

Reply:
done!